cypress/integration/Slider.spec.js

```diff
@@ -0,0 +1,83 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
```

Comment:
If you drop an extra `.eslintrc` inside the `/cypress` folder you can turn this rule off, but only for JS files in the folder. Check it out:
https://github.com/SparkPost/2web2ui/blob/master/cypress/.eslintrc

(Only recently discovered this was possible!)